dytype/test_dytype.sh

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env bash
 
+DIR=`dirname $0`
+
 run_dytype() {
-  guile --no-auto-compile dytype.scm "$@"
+  guile --no-auto-compile $DIR/dytype.scm "$@"
```

Comment:
Why `--no-auto-compile`?

Reply:
Also, when did this get switched to `guile`? I think I missed it in my conversion last night...

Reply:
Oh, I see, you already fixed me idiocy in previous commits. Thanks!

Reply:
I wonder if we should also get the tests in the developer build loop. Though in this case they wouldn't have kept me from breaking them, because I wasn't in a clean state so they would have passed anyway (and they did when I tried them).

Reply:
@Kazark when I first ran dytype it printed a long message that I can't get to come back now. `--no-auto-compile` was recommended but I don't remember why...